.gitignore

```diff
@@ -42,3 +42,1013 @@ doc/bash_completion.d
 doc/openapi.js
 doc/openapi.json
 doc/swagger
+venv/lib64
+venv/pyvenv.cfg
+venv/bin/activate
+venv/bin/activate.csh
+venv/bin/activate.fish
+venv/bin/Activate.ps1
+venv/bin/pip
+venv/bin/pip3
+venv/bin/pip3.11
+venv/bin/python
+venv/bin/python3
+venv/bin/python3.11
+venv/lib/python3.11/site-packages/_cffi_backend.cpython-311-x86_64-linux-gnu.so
+venv/lib/python3.11/site-packages/distutils-precedence.pth
+venv/lib/python3.11/site-packages/PyJWT-2.10.1.dist-info/AUTHORS.rst
+venv/lib/python3.11/site-packages/PyJWT-2.10.1.dist-info/INSTALLER
+venv/lib/python3.11/site-packages/PyJWT-2.10.1.dist-info/LICENSE
+venv/lib/python3.11/site-packages/PyJWT-2.10.1.dist-info/METADATA
```

Comment:
That ... doesnt seem super relevant to list every file in the venv individually ... x_x

Reply:
woopsies wth happened ?! i did write that, that looks like a vscode plugin doing its stuff.